src/android/TransitionValue.java

```diff
@@ -0,0 +1,23 @@
+package com.crypho.plugins;
+
+public class TransitionValue{
+    private String service, key, value;
+
+    public  TransitionValue(String service, String key, String value){
+        this.service = service;
+        this.key = key;
+        this.value = value;
+    }
+
+    public String Key() {
```

Comment:
Java method names usually start with lowercase characters. In this particular case of getters, they are usually named getVar or isVar (for booleans).